flake8_annotations/checker.py

```diff
@@ -33,6 +34,17 @@ def run(self) -> Generator[error_codes.Error, None, None]:
             for arg in function.get_missed_annotations():
                 yield classify_error(function, arg).to_flake8()
 
+    @staticmethod
+    def load_file(src_filepath: Path) -> Tuple[ast.Module, List[str]]:
```

Comment:
Is this used anywhere in this package or only for testing? If the latter, I think this should be in `testing/__init__.py`.